wsmaster/codenvy-hosted-api-license/src/test/java/com/codenvy/api/license/server/spi/tck/JpaCodenvyLicenseActionDaoTest.java

```diff
@@ -72,39 +73,76 @@ public void setUp() throws Exception {
         codenvyLicenseRepository.createAll(asList(codenvyLicenseActions));
     }
 
+
+    @AfterMethod
+    public void cleanUp() throws Exception {
+        codenvyLicenseRepository.removeAll();
+    }
+
+    @Test
+    public void shouldInsertNewRecord() throws Exception {
+        dao.insert(new CodenvyLicenseActionImpl(FAIR_SOURCE_LICENSE,
+                                                EXPIRED,
+                                                System.currentTimeMillis(),
+                                                null,
+                                                ImmutableMap.of("prop1", "value1")));
+
+        CodenvyLicenseActionImpl action = dao.getByLicenseAndAction(FAIR_SOURCE_LICENSE, EXPIRED);
+        assertNotNull(action);
+    }
+
+
     @Test(expectedExceptions = ConflictException.class)
-    public void shouldThrowExceptionIfLicenseActionAlreadyExists() throws Exception {
-        codenvyLicenseDao.store(new CodenvyLicenseActionImpl(FAIR_SOURCE_LICENSE,
-                                                             ACCEPTED,
-                                                             System.currentTimeMillis(),
-                                                             null,
-                                                             ImmutableMap.of()));
+    public void shouldThrowConflictExceptionOnInsertIfRecordExists() throws Exception {
```

Comment:
It's would be better to have test case of throwing ConflictException if we adding attribute with existed name as well.